zetta_utils/convnet/simple_inference_runner.py

```diff
@@ -15,15 +17,16 @@ class SimpleInferenceRunner:  # pragma: no cover
     model_path: str
     unsqueeze_to: int | None = None
 
-    def __call__(self, src: torch.Tensor) -> torch.Tensor:
+    def __call__(self, src: npt.NDArray) -> npt.NDArray:
         if torch.cuda.is_available():
             device = "cuda"
         else:
             device = "cpu"
 
         # load model during the call _with caching_
         model = convnet.utils.load_model(self.model_path, device=device, use_cache=True)
+
         if self.unsqueeze_to is not None:
             src = tensor_ops.unsqueeze_to(src, self.unsqueeze_to)
-        result = model(src.to(device))
+        result = model(to_torch(src).to(device))
         return result
```

Comment:
Return type (`torch.Tensor`) conflicts with the new return type definition (`npt.NDArray`).  We are a bit inconsistent in our code base. E.g. the `ModelInferencer` converts the result to the input data type, but in other locations we just always return `torch.Tensor`...